sound/soc/sof/debug-dsp-ops.c

```diff
@@ -115,5 +209,9 @@ int sof_dbg_dsp_ops_test_init(struct snd_sof_dev *sdev)
 		return ret;
 
 	/* create debugfs entry for FW path */
-	return sof_dsp_dsp_ops_create_dfse(sdev, "fw_path", dsp_ops_debugfs, 0666);
+	ret = sof_dsp_dsp_ops_create_dfse(sdev, "fw_path", dsp_ops_debugfs, 0666);
+	if (ret < 0)
+		return ret;
+
+	return sof_dsp_dsp_ops_create_dfse(sdev, "boot_fw", dsp_ops_debugfs, 0222);
```

Comment:
No control on whether we boot from IMR?

Reply:
Always a fresh boot with the boot option. I will need to add another option to do IMR boot later

Reply:
The IMR boot is the target, I don't think anyone is interested in a fresh boot, historically this created all kinds of issues in stress tests.